README.md

```diff
@@ -62,6 +62,61 @@ docker run -v /path/to/my-code:/src/main ponylang/ponyc ./main
 
 If you're using `docker-machine` instead of native docker, make sure you aren't using [an incompatible version of Virtualbox](#virtualbox).
 
+## Linux using an RPM package (via Bintray)
+
+For Red Hat, CentOS, or Fedora Linux, the `master` and `monthly` branches are packaged and availabe on Bintray ([pony-language/ponyc-rpm](https://bintray.com/pony-language/ponyc-rpm)).
+
+To install a single RPM from Bintray:
+```bash
+wget https://dl.bintray.com/pony-language/ponyc-rpm/ponyc-VERSION.rpm
+sudo rpm --install ponyc-VERSION.rpm
+yum info ponyc
+ponyc --version
+```
+
+To install monthly / release builds via Yum:
+```bash
+wget https://bintray.com/pony-language/ponyc-rpm/rpm -O bintray-pony-language-ponyc-rpm.repo
+sudo mv bintray-pony-language-ponyc-rpm.repo /etc/yum.repos.d/
+
+sudo yum install ponyc
+```
+
+To install master builds via Yum:
+```bash
+wget https://bintray.com/pony-language/ponyc-rpm/rpm -O bintray-pony-language-ponyc-rpm.repo
+sudo mv bintray-pony-language-ponyc-rpm.repo /etc/yum.repos.d/
+
+sudo yum install yum-utils
+repoquery --show-duplicates ponyc
+
+yum install ponyc-master_0.2.1_49_b15a0c5
+```
+
+That last bit of versioning is where you'll notice that we're not quite matching the name-version-release-arch pattern [described here for Fedora](https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Non-Numeric_Version_in_Release), but things should work anyways for now...
```

Comment:
As [the link I mentioned](https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Non-Numeric_Version_in_Release) describes, it'd be better to use something like this:
```
ponyc-0.2.1-0.49.master.x86_64.rpm
ponyc-0.2.1-0.51.master.x86_64.rpm
ponyc-0.2.1-0.52.master.x86_64.rpm
ponyc-0.2.1-0.53.master.x86_64.rpm
ponyc-0.2.1-0.54.master.x86_64.rpm
ponyc-0.2.1-50.release.x86_64.rpm <-- a first monthly / release build
ponyc-0.2.1-55.release.x86_64.rpm <-- a second monthly / release build
```

Perhaps it's easy to tell FPM to generate RPM files like this?